sound/soc/sof/sof-priv.h

```diff
@@ -382,6 +382,7 @@ enum snd_sof_fw_state {
 	SOF_FW_BOOT_FAILED,
 	SOF_FW_BOOT_READY_FAILED, /* firmware booted but fw_ready op failed */
 	SOF_FW_BOOT_COMPLETE,
+	SOF_FW_CRASHED,
```

Comment:
add this one and the `BOOT_READY_OK` to `fw_state_dbg`

Reply:
absolutely!

Fwiw, the fw_state_dbg and friends are trying to get merged via https://github.com/thesofproject/linux/pull/3150